.github/workflows/comments.yml

```diff
@@ -0,0 +1,27 @@
+name: comments
```

Comment:
I think we can give a more specific name. How about `Rerun trigger`?

Reply:
It's not only working for rerun github action, We can run more command from comment, just like `/ok-to-test`